beta-architecture/beta-architecture.md

```diff
@@ -10,35 +8,38 @@ This is not the final architecture, and it will change during the course of the
 
 ![](./diagrams/tdr-beta-high-level-architecture.svg)
 
-#### AWS Accounts
+## AWS Accounts and TDR environments
+
+TDR uses five AWS accounts:
 
-AWS accounts will be set up for each environment that will be used.
+* Management (mgmt): the top-level account, which is used for running
+  cross-environment services like Jenkins CI and Grafana.
+* Sandbox (sbox): used for technical spikes
+* Integration (intg): the first environment that code is deployed to. Most
+  services are deployed automatically to intg when code is merged to the
+  main/master branch. Most deployments are run programatically through Jenkins,
+  but we sometimes make temporary manual changes or deploy a branch when that's
+  the easiest way to test something.
+* Staging (staging): a more stable environment used to check changes just before
```

Comment:
Is it worth mentioning that the Staging environment is used for user testing?

Reply:
Good shout. I've added that.